cypress/integration/e2e/article.e2e.spec.js

```diff
@@ -22,6 +22,15 @@ describe('E2E Page rendering', function() {
                     cy.contains('Most popular');
                 });
 
+                if (!article.hideMostViewed) {
+                    cy.wait('@getMostReadGeo').then(xhr => {
+                        expect(xhr.response.body).to.have.property('heading');
+                        expect(xhr.status).to.be.equal(200);
+
+                        cy.contains('most viewed');
+                    });
+                }
+
```

Comment:
Might be nice to add an else test here too, as in else expect to *not* find 'most viewed'

Reply:
As discussed IRL, it's going to be trickier than expected to ensure Most Viewed is not rendered as there's a dependency on a network call that wouldn't be made.